# 9月3周

## 解决上周问题

### 1⃣️他们的标准化是怎么做的？

* 使用的scanpy的normalize_total函数

sc.pp.normalize_total(adata, target_sum=1e4)。

就是<font color='red'>（某个细胞某个基因的表达）*（target_sum）/（此细胞总共的基因表达数）</font>。

当target_sum=1e6时，就是cpm。

```
Signature: sc.pp.normalize_total(adata: anndata._core.anndata.AnnData, target_sum: Union[float, NoneType] = None, exclude_highly_expressed: bool = False, max_fraction: float = 0.05, key_added: Union[str, NoneType] = None, layers: Union[typing_extensions.Literal['all'], Iterable[str]] = None, layer_norm: Union[str, NoneType] = None, inplace: bool = True) -> Union[Dict[str, numpy.ndarray], NoneType]
Docstring:
Normalize counts per cell.

If choosing `target_sum=1e6`, this is CPM normalization.

If `exclude_highly_expressed=True`, very highly expressed genes are excluded
from the computation of the normalization factor (size factor) for each
cell. This is meaningful as these can strongly influence the resulting
normalized values for all other genes [Weinreb17]_.

Similar functions are used, for example, by Seurat [Satija15]_, Cell Ranger
[Zheng17]_ or SPRING [Weinreb17]_.

Params
------
adata : AnnData
    The annotated data matrix of shape `n_obs` × `n_vars`.
    Rows correspond to cells and columns to genes.
target_sum : typing.Union[float, NoneType], optional (default: None)
    If `None`, after normalization, each observation (cell) has a total
    count equal to the median of total counts for observations (cells)
    before normalization.
exclude_highly_expressed : bool, optional (default: False)
    Exclude (very) highly expressed genes for the computation of the
    normalization factor (size factor) for each cell. A gene is considered
    highly expressed, if it has more than `max_fraction` of the total counts
    in at least one cell. The not-excluded genes will sum up to
    `target_sum`.
max_fraction : float, optional (default: 0.05)
    If `exclude_highly_expressed=True`, consider cells as highly expressed
    that have more counts than `max_fraction` of the original total counts
    in at least one cell.
key_added : typing.Union[str, NoneType], optional (default: None)
    Name of the field in `adata.obs` where the normalization factor is
    stored.
layers : typing.Union[typing_extensions.Literal['all'], typing.Iterable[str]], optional (default: None)
    List of layers to normalize. Set to `'all'` to normalize all layers.
layer_norm : typing.Union[str, NoneType], optional (default: None)
    Specifies how to normalize layers:

    * If `None`, after normalization, for each layer in *layers* each cell
      has a total count equal to the median of the *counts_per_cell* before
      normalization of the layer.
    * If `'after'`, for each layer in *layers* each cell has
      a total count equal to `target_sum`.
    * If `'X'`, for each layer in *layers* each cell has a total count
      equal to the median of total counts for observations (cells) of
      `adata.X` before normalization.

inplace : bool, optional (default: True)
    Whether to update `adata` or return dictionary with normalized copies of
    `adata.X` and `adata.layers`.

Returns
-------
Returns dictionary with normalized copies of `adata.X` and `adata.layers`
or updates `adata` with normalized version of the original
`adata.X` and `adata.layers`, depending on `inplace`.

Example
--------
>>> from anndata import AnnData
>>> import scanpy as sc
>>> sc.settings.verbosity = 2
>>> np.set_printoptions(precision=2)
>>> adata = AnnData(np.array([
...    [3, 3, 3, 6, 6],
...    [1, 1, 1, 2, 2],
...    [1, 22, 1, 2, 2],
... ]))
>>> adata.X
array([[ 3.,  3.,  3.,  6.,  6.],
       [ 1.,  1.,  1.,  2.,  2.],
       [ 1., 22.,  1.,  2.,  2.]], dtype=float32)
>>> X_norm = sc.pp.normalize_total(adata, target_sum=1, inplace=False)['X']
>>> X_norm
array([[0.14, 0.14, 0.14, 0.29, 0.29],
       [0.14, 0.14, 0.14, 0.29, 0.29],
       [0.04, 0.79, 0.04, 0.07, 0.07]], dtype=float32)
>>> X_norm = sc.pp.normalize_total(
...     adata, target_sum=1, exclude_highly_expressed=True,
...     max_fraction=0.2, inplace=False
... )['X']
The following highly-expressed genes are not considered during normalization factor computation:
['1', '3', '4']
>>> X_norm
array([[ 0.5,  0.5,  0.5,  1. ,  1. ],
       [ 0.5,  0.5,  0.5,  1. ,  1. ],
       [ 0.5, 11. ,  0.5,  1. ,  1. ]], dtype=float32)
File:      ~/opt/miniconda3/envs/biology/lib/python3.7/site-packages/scanpy/preprocessing/_normalization.py
Type:      function
```

然后按照scanpy处理流程，跑了，得到UMAP，Zscan4c标志的细胞和之前一致，就那几个

![image-20210913104833251](https://tva1.sinaimg.cn/large/008i3skNly1guet4uhrndj61fg0p644b02.jpg)

## 2⃣️barcode看是否在多的里面

![image-20210913111358491](https://tva1.sinaimg.cn/large/008i3skNly1guetvarlh2j60gi024t8t02.jpg)

dif_bc是scTE比正常跑多出来的cell barcode；fil_z是2C-like细胞特异性表达的基因Zscan4c表达了的细胞，取其行名为表达了Zscan4c的cell barcode

两者取交集，并没有相交的，说明2C-like细胞并不在scTE多出的那些细胞里

## 3⃣️scTE自带附件文件

说明了自己的数据都是从public data repositories来的；只标注了数据来源，并没有计数矩阵

## 4⃣️整理数据处理流程发送邮件(说明复现遇到的困难；语气客气一点)

I want to reproduce finding the 2C-like small cell group experiment of mouse embryonic stem cells. 

### Download data

So I downloaded the mouse ESC data from NCBI(**[GSM3629847](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3629847)**) in sra format. 

Then I use fastq-dump in "--split-files" mode to split sra data to fastq data.

### Alignment

I use STAR 2.7.9a in solo mode to align reads.

```
STAR \
--runThreadN 16 \
--runMode alignReads \
--readFilesCommand zcat \
--genomeDir /data/tmp_data/data/yzf/reference/star_mm10 \
--readFilesIn /data/tmp_data/data/yzf/210826.ESC_2c-like/data/ESC/10x_osk_esc_S1_L008_R2_001.fastq.gz \
/data/tmp_data/data/yzf/210826.ESC_2c-like/data/ESC/10x_osk_esc_S1_L008_R1_001.fastq.gz \
--soloType CB_UMI_Simple \
--soloCBwhitelist /data/tmp_data/data/yzf/reference/star_mm10/whitelist \
--outSAMattributes NH HI AS nM CR CY UR UY \
--outFilterMultimapNmax 100 \
--winAnchorMultimapNmax 100 \
--outMultimapperOrder Random \
--runRNGseed 777 \
--outSAMmultNmax 1 \
--outSAMtype BAM Unsorted \
--outFileNamePrefix star_esc.210916
```

### Process data

```
scTE -i /data/tmp_data/data/yzf/210826.ESC_2c-like/star_esc.210916/star_esc.bam \
-x /data/tmp_data/data/yzf/210826.ESC_2c-like/scTE_ESC.210909/mm10.exclusive.idx \
-o scTE_esc_from_star \
-p 16 \
-CB CR \
-UMI UR
```

The expression of Zscan4c, the marker gene of 2C-like cell , is just like down here. The top line is its count, and the bottom is the number of cells.

![image-20210913153555962](https://tva1.sinaimg.cn/large/008i3skNly1guf1fuu1iij60m00483yp02.jpg)

I use scanpy for downstream analysis, but the result is not very good. The specific parameter settings and codes are not shown.

## ![image-20210916223447236](https://tva1.sinaimg.cn/large/008i3skNly1guiuemzfsrj60om0q6tar02.jpg)

The number of 2C-like cells is very small, and the projection shape is different.

![image-20210916223650703](https://tva1.sinaimg.cn/large/008i3skNly1guiugqhwokj30y00gyq5n.jpg)

## 问题

1. star的一个参数 --winAnchorMultimapNmax  int>0: max number of loci anchors are allowed to map to。loci anchors是啥

## PPT问题

1. 

## 老师提的意见

1. 看看最新的深度学习放进去
2. 
